package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "lint": "tfm-lint",
+    "lint:spelling": "eslint ./webpack",
```

Comment:
should this run on all the code? looks like you found a bunch of mistakes in ruby as well

Reply:
It will run only on JS file with eslint, but yeah.. I replaced this words all over ruby code too

Reply:
Running it on vanilla legacy code is a bit problematic as it also apply some ES6 linting rules..

Reply:
can we run just spell checking separately from the linting?

Reply:
Not sure if we can, any thoughts @sharvit ?

Reply:
@amirfefer can we split the eslint linting and spellcheck? so we could run spellcheck also on legacy code?